README.md

```diff
@@ -0,0 +1,177 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] This repo had no changes since the last review.  Kindly, check the comments from the last review:

1- Remove tests section (since you are not including tests)
2- Being able to create new users from the frontend (optional)
3- Correct the port in the cors.rb file in the backend, so the frontend can access.
4- Correct cors.rb to allow google drive so your images show up and are not blocked.
5- In the reservation form page, let the users go back in some way, like a back button.
6- Refactor 2 functions to ES6.
7- Touch up the "Add motorcycle" page so it fits better.
8- Improve consistency on the navbar when it's opened from the "Add motorcycle" page.